pages/hackathons/grant.js

```diff
@@ -56,15 +56,14 @@ const Requirement = ({ title, children, checkmark, background, size }) => {
 }
 
 const HackathonGrant = () => {
-  let open = true // applications are open
-  let channel = 'https://hackclub.slack.com/archives/C03TS0VKFPZ' // #hackathon-grants
+  let open = false // applications are open
```

Comment:
this is no longer used

```suggestion
```

Reply:
[`55b7ce7` (#1446)](https://github.com/hackclub/site/pull/1446/commits/55b7ce71b0f60dc27a7fcfe7668b3cc73a4bc26d)